docugen/generate.py

```diff
@@ -239,7 +258,25 @@ def write_docs(
 
             content = []
             content.append(pretty_docs.build_md_page(page_info))
-            text = markdownify.markdownify("\n".join(content), escape_underscores=False)
+
+            # Clean up markdown and remove characters that break Docusuarus
+            dictionary = {
+                "<" : "",
```

Comment:
should this and `>` be replaced by `&lt;` and `&gt;` respectively?  Looks like that works in MD: &lt; &gt;

Reply:
We'll keep that out for now. They might return once we define an API style guide. But for now, i'll keep it uniform with the rest of the API docs.